crates/kernel/testsuite/moot/eval/simple.moot

```diff
@@ -1,30 +1,30 @@
-## `eval` takes only string arguments
+// `eval` takes only string arguments
 
 ; return eval(1);
 E_TYPE
 
-# TODO: uncomment once our eval supports multiple arguments
-#; return eval(1, 2);
-#E_TYPE
+// TODO: uncomment once our eval supports multiple arguments
```

Comment:
Is multiple arguments a thing that stunt supports? LambdaMOO 1.8 believes in only 1.

Reply:
Oops. I'll drop the references multi-args then :D